zookeeper-contrib/zookeeper-contrib-zkpython/src/c/zookeeper.c

```diff
@@ -1477,8 +1491,11 @@ PyObject *pyzoo_set_log_stream(PyObject *self, PyObject *args)
   }
   
 #if PY_MAJOR_VERSION >= 3
-  extern PyTypeObject PyIOBase_Type;
-  if (!PyObject_IsInstance(pystream, (PyObject *)&PyIOBase_Type)) {
+  if (init_file_emulator() < 0) {
+    return NULL;
+  }
```

Comment:
I'm not sure about it's at the right place. Init should only be called once, but now it will be called every time somebody sets the log stream, which usually happens once, still must be a better place for this.